src/core/service/notifications/jobs/handler.go

```diff
@@ -92,27 +93,36 @@ func (h *Handler) Prepare() {
 
 // HandleScan handles the webhook of scan job
 func (h *Handler) HandleScan() {
-	log.Debugf("received san job status update event: job-%d, status-%s, track_id-%s", h.id, h.status, h.trackID)
+	log.Debugf("received san job status update event: job UUID: %s, status-%s, track id-%s", h.change.JobID, h.status, h.trackID)
 
 	// Trigger image scan webhook event only for JobFinished and JobError status
 	if h.status == models.JobFinished || h.status == models.JobError {
-		e := &event.Event{}
-		metaData := &event.ScanImageMetaData{
-			JobID:  h.id,
-			Status: h.status,
-		}
-		if err := e.Build(metaData); err == nil {
-			if err := e.Publish(); err != nil {
-				log.Errorf("failed to publish image scanning event: %v", err)
-			}
+		// Get the required info from the job parameters
+		req, err := sc.ExtractScanReq(h.change.Metadata.Parameters)
+		if err != nil {
+			log.Error(errors.Wrap(err, "scan job hook handler: event publish"))
 		} else {
-			log.Errorf("failed to build image scanning event metadata: %v", err)
+			e := &event.Event{}
+			metaData := &event.ScanImageMetaData{
+				Artifact: req.Artifact,
+				Status:   h.status,
+			}
+
+			if err := e.Build(metaData); err == nil {
```

Comment:
This is trivial but generally we should avoid nested `if`
```
if err:= e.Build(); err!= nil {
    ..
    return
}
if err:= e.Publish(); err != nil {
    ... 
   return
}
```
is a better approach